regtests/ww3_tp2.15/input/ww3_ounp_spec.nml

```diff
@@ -77,6 +77,7 @@
 !                                          ! 4: Spectral partitioning
 !     SPECTRA%SCALE_FAC     = 1            ! Scale factor (-1=disabled)
 !     SPECTRA%OUTPUT_FAC    = 0            ! Output factor (0=normalized)
+!     SPECTRA%TYPE          = 4            ! [2 = SHORT, 3 = it depends , 4 = REAL]
```

Comment:
Same here as above.

Reply:
Totally agree, now this line was deleted.